### PR TITLE
Add barTrackInverse token

### LINKED ISSUE #1655: → Inverse | Progress bar
- [x] Update specs
- [ ] Design documentation
- [x] [Add to mobile library](https://www.figma.com/design/WCkDDzlXE16R6yXaljxddj/branch/HRjIjMZr0wBbZZyiVohuYr/M%C3%ADstica-Mobile?m=auto&t=emiGOhm9p16Dkr7K-1)
- [x] [Add to desktop library](https://www.figma.com/design/DSWhPLyJzbliP1fBrLxDUR/branch/TMvB9XsnSp2E92BJHcvEnk/M%C3%ADstica-Desktop?m=auto&t=ROZqEHCUTG8WPa2Y-1)
- [ ] Add to UI Kit
- [x] [Android ticket](https://jira.tid.es/browse/ANDROID-15310)
- [x] [iOS ticket](https://jira.tid.es/browse/IOS-10643)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2084)

### Describe the bug

In the CMS, any module can be represented in 3 possibles backgrounds, and when used in brand background, the progress bar "dissapears"
imagen (1)

### What libraries are you seeing the problem on?

Mobile Library

### Steps to reproduce

New CMS module, still not possible to redirect to an URL

### Expected behavior

The progress bar should have a inverse behavior. Here are some ideas, but I know you will find a best definition :)
image

### PR DESCRIPTION
**Pull request by aweell**
Tokens preview: https://mistica-design.vercel.app/tokens-map/aweell-add-barTrackInverse/blau/color/barTrackInverse/

Color study: https://www.figma.com/design/3IO1EsGDeBRiFdF23mrvuj/branch/Ru8qXVDslEVEOUWjQZEexZ/%F0%9F%94%B8-ProgressBar-Specs?m=auto&node-id=2602-338&t=OrXNcFAXv8z510Pf-1

### CODE REVIEW

File: tokens/movistar.json [hunk 1]
```diff
@@ -335,6 +335,11 @@
       "type": "color",
       "description": "grey3"
     },
+    "barTrackInverse": {
+      "value": "rgba({palette.black}, 0.14)",
```
Comment (yceballost) [inline tokens/movistar.json:339]: 0.14, why this number? xD

File: tokens/tu.json [hunk 2]
```diff
@@ -335,6 +335,11 @@
       "type": "color",
       "description": "grey2"
     },
+    "barTrackInverse": {
+      "value": "rgba({palette.white}, 0.36)",
```
Comment (yceballost) [inline tokens/tu.json:339]: same for 0.36, it seems strange to me that the numbers are so specific

**Review by yceballost — APPROVED** [review]

File: tokens/movistar.json (hunk 1 above)
Comment (aweell) [reply tokens/movistar.json:339]: Is the result of finding the nearest contrast ratio to the actual in default context using a black with opacity over background color.

**Review by marinitx — APPROVED** [review]

File: tokens/o2.json
Comment (andredelramo-hm) [inline tokens/o2.json:1]: Seguimos manteniendo O2?

Comment (yceballost) [reply tokens/o2.json:1]: yep, we need to keep this brand until code remove it

**Review by andredelramo-hm — APPROVED** [review]
good

**Pull request merged**